tests/sentry/event_manager/interfaces/snapshots/test_spans/test_multiple_full.pysnap

```diff
@@ -1,13 +1,11 @@
 ---
-created: '2019-07-11T19:30:21.322813Z'
-creator: sentry
 source: tests/sentry/event_manager/interfaces/test_spans.py
 ---
 errors: null
 to_json:
 - data:
+    http.response.status_code: 200
```

Comment:
Relay renamed the value for this key. It seems it didn't have a negative impact in other places. (same for the snapshot below)